WeakAuras/Conditions.lua

```diff
@@ -384,110 +380,116 @@ local function ParseProperty(property)
+          if debug then
+            ret = ret .. "    print('- " .. change.property .. " " .. formatValueForAssignment(propertyData.type, GetBaseProperty(data, change.property)) .. "')\n"
+          end
         end
       end
     end
     ret = ret .. "  end\n"
   end
 
-  return ret;
+  return ret
 end
 
 local function CreateActivateCondition(ret, id, condition, conditionNumber, properties, debug)
-  if (condition.changes) then
+  if condition.changes then
     ret = ret .. "  if (newActiveConditions[" .. conditionNumber .. "]) then\n"
-    ret = ret .. "    if (not activatedConditions[".. conditionNumber .. "]) then\n"
-    if (debug) then ret = ret .. "      print('Activating condition " .. conditionNumber .. "' )\n"; end
+    ret = ret .. "    if (not activatedConditions[" .. conditionNumber .. "]) then\n"
+    if debug then
+      ret = ret .. "      print('Activating condition " .. conditionNumber .. "' )\n"
+    end
     -- non active => active
     for changeNum, change in ipairs(condition.changes) do
-      if (change.property) then
+      if change.property then
         local propertyData = properties and properties[change.property]
-        if (propertyData and propertyData.type) then
-          if (propertyData.setter) then
-            ret = ret .. "      propertyChanges['" .. change.property .. "'] = " .. formatValueForAssignment(propertyData.type, change.value) .. "\n";
-            if (debug) then ret = ret .. "      print('- " .. change.property .. " " .. formatValueForAssignment(propertyData.type, change.value) .. "')\n"; end
-          elseif (propertyData.action) then
-            local pathToCustomFunction = "nil";
+        if propertyData and propertyData.type then
+          if propertyData.setter then
+            ret = ret .. "      propertyChanges['" .. change.property .. "'] = " .. formatValueForAssignment(propertyData.type, change.value) .. "\n"
+            if debug then
+              ret = ret .. "      print('- " .. change.property .. " " .. formatValueForAssignment(propertyData.type, change.value) .. "')\n"
+            end
+          elseif propertyData.action then
+            local pathToCustomFunction = "nil"
             local pathToFormatter = "nil"
-            if (WeakAuras.customConditionsFunctions[id]
-              and WeakAuras.customConditionsFunctions[id][conditionNumber]
-              and  WeakAuras.customConditionsFunctions[id][conditionNumber].changes
-              and WeakAuras.customConditionsFunctions[id][conditionNumber].changes[changeNum]) then
-              pathToCustomFunction = string.format("WeakAuras.customConditionsFunctions[%q][%s].changes[%s]", id, conditionNumber, changeNum);
+            if WeakAuras.customConditionsFunctions[id] and WeakAuras.customConditionsFunctions[id][conditionNumber] and WeakAuras.customConditionsFunctions[id][conditionNumber].changes and WeakAuras.customConditionsFunctions[id][conditionNumber].changes[changeNum] then
```

Comment:
the problem with setting the line width to 1000 is that stylua thinks this is what we want.

Obviously we gain a bit of consistency by removing all the semicolons, fixing the indentation and spaces around operators.

But this and other lines that are now exessively long are not an improvement.

Reply:
Yeah but that's when we can use the inline ignore, doesn't happen too often?

Reply:
Well, that's just one of many examples. And frankly the gain is just not there, it fixes a few small things everywhere, but looking at what it does, I haven't found a clear example of where the existing formatting was actually unreadable.

As in, I haven't seen anywhere where the indentation was actually misleading.

I'd much rather have a tool that respects the developer instead of this by design user hostile tool.